### ISSUE TITLE
test new dataset

### ISSUE THREAD

**Issue by ke-s** [zwx8981/UNIQUE#9]
Hello, may I ask, if I want to test the quality of the image on the coco dataset,Can I test the coco data set directly with the pre-training model you gave，direct the path of the test picture in the demo to the coco dataset，or need to train the model from scratch?
Thank you very much for your reply

**Comment by zwx8981** [comment]
@ke-s  If you have the MOS/DMOS labels for the new dataset, you can either directly test it using the pre-trained model or train another model from scratch with your dataset.  Otherwise, you can still use the pre-trained model to output a quality score for any given test image on your new dataset, but you cannot measure the quality prediction performance due to the absence of ground-truth labels. Thank you.
